projects/akita-ng-fire/src/lib/utils/sync-from-action.ts

```diff
@@ -10,6 +10,15 @@ export function setLoading(storeName: string, loading: boolean) {
   });
 }
 
+/** Set the loading parameter of a specific store */
```

Comment:
I think the comment is not related to the function

Reply:
you're right, I fixed this mistake